src/appsettings.json

```diff
@@ -13,7 +13,7 @@
     "ChatContainer": "chat",
     "CacheContainer": "cache",
     "ProductContainer": "products",
-    "ProductDataSource": "https://cosmosdbcosmicworks.blob.core.windows.net/cosmic-works-vectorized/product-text-3-large-1536.json"
+    "ProductDataSourceURI": "https://cosmosdbcosmicworks.blob.core.windows.net/cosmic-works-vectorized/product-text-3-large-1536.json"
```

Comment:
Can you update the azure.yml file that gets updated with the updated secrets after deployment to include this updated product data source uri?